ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -100,15 +100,17 @@ static void setElementLimit(RaftProperties properties, int limit) {
       setInt(properties::setInt, ELEMENT_LIMIT_KEY, limit, requireMin(1));
     }
 
-    String BYTE_LIMIT_KEY = PREFIX + ".byte-limit";
-    SizeInBytes BYTE_LIMIT_DEFAULT = SizeInBytes.valueOf("64MB");
-    static SizeInBytes byteLimit(RaftProperties properties) {
-      return getSizeInBytes(properties::getSizeInBytes,
-          BYTE_LIMIT_KEY, BYTE_LIMIT_DEFAULT, getDefaultLog());
+    String MEGA_BYTE_LIMIT_KEY = PREFIX + ".megabyte-limit";
```

Comment:
We cannot rename the key.  It is an incompatible change.

Reply:
done.